seed/analysis_pipelines/better/pipeline.py

```diff
@@ -300,7 +301,32 @@ def _process_results(self, analysis_id, progress_data_key):
     progress_data = ProgressData.from_key(progress_data_key)
     progress_data.step('Processing results')
 
-    # create the results Columns if the don't already exist
+    # store all measure recommendations
+    ee_measure_names = [
+        'Upgrade Windows',
+        'Reduce Plug Loads',
+        'Add/Fix Economizers',
+        'Decrease Ventilation',
+        'Reduce Lighting Load',
+        'Check Fossil Baseload',
+        'Decrease Infiltration',
+        'Decrease Heating Setpoints',
+        'Eliminate Electric Heating',
+        'Increase Cooling Setpoints',
+        'Reduce Equipment Schedules',
+        'Add Wall/Ceiling Insulation',
+        'Increase Cooling System Efficiency',
+        'Increase Heating System Efficiency'
+    ]
+    ee_measure_column_data_paths = [
+        ExtraDataColumnPath(
+            f'better_recommendation_{ee_measure_name.lower().replace(" ", "_")}',
+            f'BETTER Recommendation: {ee_measure_name}',
+            f'assessment.ee_measures.{ee_measure_name}'
```

Comment:
looks good. we'll eventually want to map these to the Measures world in SEED/BSync, but good for now.

Reply:
👍 Created this ticket to track: https://github.com/SEED-platform/seed/issues/2804